computeparticles/computeparticles.cpp

```diff
@@ -58,7 +59,7 @@ class VulkanExample : public VulkanExampleBase
 		float deltaT;
 		float destX;
 		float destY;
-		int32_t particleCount = PARTICLE_COUNT;
+		uint32_t particleCount = PARTICLE_COUNT;
```

Comment:
later used in paticle.comp and compared to a uint, particleCount should never be negative.